rllib/connectors/connector.py

```diff
@@ -149,59 +149,63 @@ class AgentConnector(Connector):
     AgentConnectorDataTypes can be used to specify arbitrary type of env data,
 
     Example:
-    .. code-block:: python
-        # A dict of multi-agent data from one env step() call.
-        ac = AgentConnectorDataType(
-            env_id="env_1",
-            agent_id=None,
-            data={
-                "agent_1": np.array(...),
-                "agent_2": np.array(...),
-            }
-        )
+        .. testcode::
+            import numpy as np
```

Comment:
Rather than having one large testcode, I'd recommend splitting this into multiple `testcode` for better readability:

```
        Represent a list of agent data from one env step() call.
 
        .. testcode::

            import numpy as np
            ac = AgentConnectorDataType(
                env_id="env_1",
                agent_id=None,
                data={
                    "agent_1": np.array([1, 2, 3]),
                    "agent_2": np.array([4, 5, 6]),
                }
            )

        Or a single agent data ready to be preprocessed.

        .. testcode::

            ac = AgentConnectorDataType(
                env_id="env_1",
                agent_id="agent_1",
                data=np.array([1, 2, 3]),
            )

        etc.
```